web/flux_to_redux/advanced/examples/influx_implementation/components/redux_big_block.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import '../store.dart';
+import '../../../components/little_block.dart';
+
+part 'redux_big_block.over_react.g.dart';
+
+// When refactoring from a Flux component to a Redux component (with multiple stores):
+// [1] Add props for the state values that are needed, and remove the store prop.
+// [2] Add props for the actions that are needed (if you intend to use mapDispatchToProps).
+// [3] Refactor calls to the `store` prop to the new props added in [1] and [2].
+// [4] Make a `composeHocs` call, passing in a `List`.
+// [5] Make a `connect` call for each store the component needs access to, with each call being an element in the `List`.
+// [6] Tie `connect` to the correct context by using the `context` parameter.
+// [7] Use `mapStateToProps` and `mapDispatchToProps` to point props to the correct state values.
+// [8] Set the `pure` constructor on `connect` to be `false`.
+
+/*[4]*/ UiFactory<ReduxBigBlockProps> ConnectedReduxBigBlock = composeHocs([
+  /*[5]*/ connect<RandomColorStore, ReduxBigBlockProps>(
+    /*[6]*/ context: randomColorStoreContext,
+    /*[7]*/ mapStateToProps: (state) => (ReduxBigBlock()
```

Comment:
The way these comments affect indentation makes it hard to see the visual hierarchy/nesting of things. Thoughts on putting comments on their own lines or at the end of the lines, like so? https://gist.github.com/greglittlefield-wf/deb3f9eeca051aa46284080058f13d10

Reply:
Yep, I like that a lot better!! I'm happy to go through and do that for other instances if it's easier, or happy to accept any PRs to fix it. Just let me know! :)

Reply:
I'll make a PR, thanks!